### PR TITLE
Teams, Managers, and General Milestones

### PR DESCRIPTION
**Pull request by Markoozy**
Organized Team into advanced topic subteams, organized managers, wrote game description, started on General Milestones.

### CODE REVIEW

File: management/PuzzleStealth.md [hunk 1]
```diff
@@ -1,76 +1,76 @@
...
+	* Manager: Marko Djurdjevic
 	* Goals:
-		1. GOAL1
-		1. GOAL2
-		1. GOAL3
+		A. Set up organization repo
+		B. Organize advanced topic subteams
+		C. Lay out the goals and road map for term
 		...
-1. 9/9 - 9/15
```
Comment (nfarnan) [inline management/PuzzleStealth.md:10]: Markdown generally doesn't care what the numbers are, e.g., leaving them all 1. will still render as 1., then 2., etc.
No need to change, just FYI
https://guides.github.com/features/mastering-markdown/

File: teams/PuzzleStealth.md [hunk 2]
```diff
@@ -2,48 +2,111 @@
 
 ## Canonical game repo URL:
 
-URLGoesHere
+https://github.com/1666-Puzzle-Stealth/StealthPuzzle
```
Comment (nfarnan) [inline teams/PuzzleStealth.md:5]: I can't view this repo, please add me as a member so that I can access it.

File: teams/PuzzleStealth.md [hunk 3]
```diff
@@ -2,48 +2,111 @@
...
-* Advanced Topic Subteam 2 (TopicGoesHere)
-	* StudentName
-		* Pitt ID:
-		* Github Username:
+
+	* Anna Beach
+		* Pitt ID: arb223
+		* Github Username: arbeach123
+		* Personal Fork URL:
+
```
Comment (nfarnan) [inline teams/PuzzleStealth.md:34]: The advanced topic teams need to be as closely balanced as possible. One student will need to move from AI to Level Gen.

File: teams/PuzzleStealth.md [hunk 4]
```diff
@@ -2,48 +2,111 @@
...
+Some puzzles will require the Agent and the Hacker to work together and exchange information in order to succeed.
+
+Objectives for the team to complete will range from extracting VIPs, stealing sensitive information, assassinating a target, etc.
 
 ## General Milestones
 
-* Milestone1
-* Milestone2
-* Milestone3
+* Agent can move around in an environment
```
Comment (nfarnan) [inline teams/PuzzleStealth.md:83]: Size of environment? General scale? It's fine to move this question to the level gen milestone.

File: teams/PuzzleStealth.md [hunk 5]
```diff
@@ -2,48 +2,111 @@
...
+* Hacker can see simplified representation of facility
+* Hacker has multiple windows
+* Hacker can access subsystems
```
Comment (nfarnan) [inline teams/PuzzleStealth.md:86]: How many is the goal? Examples?

File: teams/PuzzleStealth.md [hunk 6]
```diff
@@ -2,48 +2,111 @@
...
+* Agent can give Hacker access to systems
+* Both players have puzzles to solve to access systems
```
Comment (nfarnan) [inline teams/PuzzleStealth.md:88]: Balancing the creation of puzzles with random level gen will be tricky. Any examples of the type of puzzles?

File: teams/PuzzleStealth.md [hunk 7]
```diff
@@ -2,48 +2,111 @@
...
+* Collaborative puzzles
+* Agent can interact with guards (KO, kill, etc).
+
```
Comment (nfarnan) [inline teams/PuzzleStealth.md:91]: What enemy types will exist? Just guards, all the same type? What is their expected behavior (will tie in with AI milestones)?

File: management/PuzzleStealth.md [hunk 8]
```diff
@@ -1,76 +1,76 @@
 # CS1666 PuzzleStealth Managment weeks
 
 1. 9/2 - 9/8
-	* Manager:
+	* Manager: Marko Djurdjevic
```
Comment (nfarnan) [inline management/PuzzleStealth.md:4]: I needed your sign-up by Wednesday, so this part is late.

**Review by nfarnan — CHANGES_REQUESTED** [review]

File: teams/PuzzleStealth.md (hunk 2 above)
Comment (Markoozy) [reply teams/PuzzleStealth.md:5]: I have invited you to the Puzzle Stealth Organization

File: management/PuzzleStealth.md (hunk 8 above)
Comment (Markoozy) [reply management/PuzzleStealth.md:4]: I didn't realize. I just thought we needed to tell you which of us was the manager, which I did during class. My bad.

File: teams/PuzzleStealth.md (hunk 3 above)
Comment (Markoozy) [reply teams/PuzzleStealth.md:34]: Erik Houseworth has been moved to Level Generation

File: teams/PuzzleStealth.md (hunk 5 above)
Comment (Markoozy) [reply teams/PuzzleStealth.md]: Fixed

File: teams/PuzzleStealth.md (hunk 6 above)
Comment (Markoozy) [reply teams/PuzzleStealth.md]: Gave examples.

File: teams/PuzzleStealth.md (hunk 7 above)
Comment (Markoozy) [reply teams/PuzzleStealth.md:92]: Specified enemy types/states.

File: teams/PuzzleStealth.md (hunk 4 above)
Comment (Markoozy) [reply teams/PuzzleStealth.md]: Specified. Grid based system for level creation.

**Review by nfarnan — APPROVED** [review]

**Pull request merged**